.dir-locals.el

```diff
@@ -0,0 +1,5 @@
+((coq-mode . ((eval . (let* ((project-root (locate-dominating-file buffer-file-name "_CoqProject"))
+                             (coqutil-folder (expand-file-name "../coqutil/src" project-root))
+                             (coq-path (split-string (or (getenv "COQPATH") "") path-separator t)))
+                        (unless (member coqutil-folder coq-path)
+                          (setenv "COQPATH" (mapconcat #'identity (cons coqutil-folder coq-path) path-separator))))))))
```

Comment:
I have a `.dir-locals.el` in the toplevel `bedrock2` directory, and it seems that this new `.dir-locals.el` shadows the `.dir-locals.el` in the toplevel `bedrock2` directory. @JasonGross what's the best way to deal with this?

Reply:
Shouldn't it not matter?  What's in the bedrock2 `.dir-locals.el` which you need not just when working on bedrock2, but also when writing on the files in coqutil?

Reply:
My toplevel `.dir-locals.el` contains
```
((nil . ((compile-command . "if git rev-parse --is-inside-work-tree ; then cgexec -g memory:myMemGroup make all -C `git rev-parse --show-superproject-working-tree --show-toplevel | head -1` -j -k ; SAVED_EXIT_CODE=$? ; if test $SAVED_EXIT_CODE = 0 ; then notify-send 'make' 'compilation succeeded'; else notify-send 'make' 'compilation failed' ; exit $SAVED_EXIT_CODE ; fi ; else echo 'not sure what to do' ; fi"))))
```

And I also have a shortcut for the emacs compile command, so no matter in which submodule I am, I can rerun `make` for the toplevel bedrock2 project with just one keystroke, a feature I need and use very often.

Reply:
Perhaps @cpitclaudel can comment on having multiple `.dir-locals.el` files, but what about putting those commands in your `~/.emacs` file?  (Note that I'll be adding a `.dir-locals.el` to bedrock2 to set bedrock2's `COQPATH` appropriately.)